app/models/user.rb

```diff
@@ -67,11 +75,28 @@ def password_reset_expired?
   end
 
   def feed
-    Micropost.where("user_id = ?", id)
+    following_ids = "SELECT followed_id FROM relationships
+                     WHERE follower_id = :user_id"
+    Micropost.where("user_id IN (#{following_ids})
+                     OR user_id = :user_id", user_id: id)
```

Comment:
```suggestion
    Micropost.where(user_id: following_ids).or(Micropost.where(user_id: id))
```

FYI
よりシンプルに`or`を用いて上記のような書き方もできます。
https://style.potepan.com/articles/29804.html